_pkgdown.yml

```diff
@@ -0,0 +1,11 @@
+url: ~
+
+reference:
+- title: Main functions
+  contents:
+  - has_concept("main functions")
+- title: Helper functions
+  contents:
+  - has_concept("helpers")
+  - lookup
```

Comment:
The setup of the website seems to belong to a separate PR. I may do that then fix the merge conflicts.